### PR TITLE
Stabilizing nnscore code.

### PR DESCRIPTION
**Pull request by rbharath**
Stabilizes the nnscore code:

-) The implementation can now successfully featurize the pdbbind core set. This PR includes a number of bugfixes that made this featurization possible, along with a collection of featurization and preprocessing-scripts. A number of extra test-cases were added as well.
-) The nnscore codebase now passes pylint.

### CODE REVIEW

File: vs_utils/scripts/public_data/nnscore_utilities.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+"""
```
Comment (skearnes) [inline vs_utils/scripts/public_data/nnscore_utilities.py]: This should be in utils.

**Comment by skearnes** [comment]
Are some of your test data files created with AutoDockTools? I think it would be best to only include data generated with OpenBabel.

**Comment by skearnes** [comment]
This looks good on a high level; I didn't do a thorough review. Just a couple of comments to address.

File: vs_utils/scripts/public_data/nnscore_utilities.py (hunk 1 above)
Comment (rbharath) [inline vs_utils/scripts/public_data/nnscore_utilities.py]: Done.

**Comment by rbharath** [comment]
My test data files come from either DUD-E and PDBBind and are processed with my openbabel pipeline. The one exception is c-Abl which comes from the autodock vina tutorial (I think there shouldn't be a problem with including this file since it's in the public domain).

**Comment by rbharath** [comment]
Thanks for the quick review. I've added in a few more tests (for bugs I caught today), and I've made the requested changes.

PTAL.

**Comment by skearnes** [comment]
Can you push your latest commit? I don't seen any of your changes. Also, please add a comment to the data files you didn't generate or that are similarly derived giving their source.

**Comment by rbharath** [comment]
Oops, sorry, pushed now.

Sources of all test files are documented in nnscore_test.py.

**Comment by skearnes** [comment]
LGTM.

**Pull request merged**